code/modules/projectiles/guns/projectile.dm

```diff
@@ -124,10 +124,7 @@
 
 /obj/item/weapon/gun/projectile/process_chambered()
 	var/obj/item/ammo_casing/AC = getAC()
-	if(in_chamber)
-		return 1 //{R}
-	if(isnull(AC) || !istype(AC))
-		return
+	can_discharge()
```

Comment:
Okay, what's a proper way to fix it? `if can_discharge() then return can_discharge()`?